### PR TITLE
Feature-1 for Review

### PR DESCRIPTION
**Pull request by Felipeg005**
In this branch next actions were taken:

1. Include my works section (titles, cards, programs used, and buttons)
2. Backgrounds and design
3. About section with skills boxes.
4. Everything according to the template 3.

Please visualize it in responsive mode. 

Thank you! :grinning:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,13 +16,123 @@
...
+        <div class="cards card1">
+            <div class="cardimg cardimg1"><img src="./Images/transparent-img.png" alt="cardimg1" height="250" width="100"/></div>
+            <h3 class="cardtitle">Multi-Post Stories<br>Gain+Glory</h3>
+            <ul class="proglist">
+                <li class="prg1">Ruby on rails</li>
+                <li class="prg2">css</li>
+                <li class="prg3">JavScript</li>
+                <li class="prg4">html</li>
+            </ul>
+            <button type="button" class="cardbutton cardbutton1">See Project</button>
```
Comment (Georjane) [inline index.html:39]: - [x] For each project, the button (e.g., "See project") is a `button` tag that you used correctly. Can you please also follow the button styling: Active, Hover, and Pressed (see Figma design below). This makes your User Experience better.
image

**Review by Georjane — CHANGES_REQUESTED** [review]
### Hello, Team @Felipeg005, ✋🏽 

Good job so far! Your portfolio is coming out perfectly 💯. You have a very professional Readme file, your PR is professional and you have a good mastery of git-flow. ✔️ 
Your linters are all passing, I applaud that too. ✔️ 
However, we have a few more changes to make in order to meet all requirements for this project but you are almost there!

## Required Changes ♻️

- [x] Kindly follow the button styling: Active, Hover, and Pressed for all buttons on your page ([see Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468))

_Check the comment under the review._

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Felipeg005) [reply index.html:39]: Oh, that's right, I forgot it :scream:, but now it's already done, thank you for your review :blush:.

**Review by Emmyn5600 — APPROVED** [review]
## Status: Approved :heavy_check_mark: :muscle::clap:

Hi @Felipeg005 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it :+1:.
Congratulations! :tada:

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**